README.md

```diff
@@ -39,23 +49,44 @@ ethane.save('ethane.top')
 ethane.save('ethane.gro')
 ```
 
-If you use this package, please cite [our paper](https://arxiv.org/abs/1812.06779), which is currently a pre-print.
+## Getting started
 
-### Getting started?
-Check out our example template for disseminating force fields:
-https://github.com/mosdef-hub/forcefield_template
+#### Getting started with SMARTS-based atom-typing
+* [SMARTS-based atomtyping](docs/smarts.rst)
+* [Supported SMARTS Grammar](https://github.com/mosdef-hub/foyer/issues/63)
 
-### [Installation instructions](docs/installation.rst)
+#### Defining force fields: 
+* [Defining force field parameters](docs/parameter_definitions.md)
+* [Force field file validation](docs/validation.rst)
+
+ 
+#### Example foyer force field files:
+Foyer currently includes a subset of the OPLS AA and TraPPE forcefields, currently part of the source distribution:
+* https://github.com/mosdef-hub/foyer/tree/master/foyer/forcefields
+
+Additional example force field XML files:
+* https://github.com/chrisiacovella/OPLSaa_perfluoroalkanes
+* https://github.com/mosdef-hub/forcefield_perfluoroethers
+* https://github.com/summeraz/OPLSaa_alkylsilanes
+
+Example template for disseminating force fields:
+* https://github.com/mosdef-hub/forcefield_template
 
-### [SMARTS-based atomtyping](docs/smarts.rst)
-* [Supported SMARTS Grammar](https://github.com/mosdef-hub/foyer/issues/63)
 
-### [Force field validation](docs/validation.rst)
+#### Using Foyer to perform atom typing:
+* [Basic usage examples](docs/usage_examples.rst)
+* [Detailed Jupyter notebook tutorials, including integration with mBuild](https://github.com/mosdef-hub/foyer_tutorials)
+* [Jupyter notebook tutorials, from [our paper](https://arxiv.org/abs/1812.06779)]()
 
-### [Defining force field parameters](docs/parameter_definitions.md)
+### Documentation site:
+* Mirror of github doc pages: http://mosdef-hub.github.io/foyer/
```

Comment:
Is this really a mirror?

Reply:
Yeah, i dont know if mirror is the operative term here. 
maybe
```suggestion
* Documentation website: http://mosdef-hub.github.io/foyer/
```

Reply:
Yeah, I'm pretty sure the content is exactly the same and just generated from the stuff in doc.